lib/IDE/CodeCompletionResultBuilder.h

```diff
@@ -255,7 +255,7 @@ class CodeCompletionResultBuilder {
                      DeclAttrParamKeyword, Name);
     if (NeedSpecify)
       addChunkWithText(CodeCompletionString::Chunk::ChunkKind::
-                       DeclAttrParamEqual, "=");
+                       DeclAttrParamEqual, ": ");
```

Comment:
Yes, please!  And please update the doc comment above its declaration in CodeCompletion.h.